docs/learn/advanced/08-events.md

```diff
@@ -46,18 +45,18 @@ Lastly, Events are returned to the underlying consensus engine in the response o
 * [`BeginBlock`](./00-baseapp.md#beginblock)
 * [`EndBlock`](./00-baseapp.md#endblock)
 * [`CheckTx`](./00-baseapp.md#checktx)
-* [`Transaction Execution`](./00-baseapp.md#transactionexecution)
+* [`Transaction Execution`](./00-baseapp.md#transaction-execution)
 
 ### Examples
 <!-- markdown-link-check-disable -->
 The following examples show how to query Events using the Cosmos SDK.
 
-| Event                                            | Description                                                                                                                                              |
-| ------------------------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `tx.height=23`                                   | Query all transactions at height 23                                                                                                                      |
-| `message.action='/cosmos.bank.v1beta1.Msg/Send'` | Query all transactions containing a x/bank `Send` [Service `Msg`](../../build/building-modules/03-msg-services.md). Note the `'`s around the value.                  |
-| `message.module='bank'`                          | Query all transactions containing messages from the x/bank module. Note the `'`s around the value.                                                       |
-| `create_validator.validator='cosmosval1...'`     | x/staking-specific Event, see [x/staking SPEC](../../build/modules/staking/README.md).                                                         |
+| Event                                            | Description                                                                                                                                          |
+| ------------------------------------------------ |------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `tx.height=23`                                   | Query all transactions at height 23                                                                                                                  |
+| `message.action='/cosmos.bank.v1beta1.Msg/Send'` | Query all transactions containing an x/bank `Send` [Service `Msg`](../../build/building-modules/03-msg-services.md). Note the `'`s around the value. |
+| `message.module='bank'`                          | Query all transactions containing messages from the x/bank module. Note the `'`s around the value.                                                   |
+| `create_validator.validator='cosmosval1...'`     | x/staking-specific Event, see [x/staking SPEC](../../../x/staking/README.md#events).                                                                 |
```

Comment:
ditto

Reply:
yeah this should be reverted, it will resolve on the website, but not on here